### PR TITLE
Update dUNL facts & rippled's transfer

### PR DESCRIPTION
**Pull request by wojake**
Ripple contributes to rippled, the XRPLF has taken over ownership of rippled whilst many diverse maintainers & contributors are still contributing.

### CODE REVIEW

File: content/faq.md [hunk 1]
```diff
@@ -101,6 +101,12 @@ Publishers of individual default UNLs set their own policies for when to add or
 For recommendations and best practices, see [Run `rippled` as a Validator](run-rippled-as-a-validator.html).
 
 
+#### If the dUNL has the most influence on the network, then is the XRPL centralized?
+False, validators can choose to not listen to the dUNL or any widely used UNLs for that matter. They can can create a brand new UNL at anytime.
```
Comment (intelliot) [inline content/faq.md]:
```suggestion
Validators can choose to not use the dUNL, or any widely-used UNL for that matter. Anyone can create a new UNL at any time.
```

File: content/faq.md [hunk 2]
```diff
@@ -101,6 +101,12 @@ Publishers of individual default UNLs set their own policies for when to add or
...
+
+We're also not taking into account of the other UNLs on the network, which also has a huge say in ledger progress. The dUNL is simply the most widely used UNL out there, and validators can choose to opt-out and use another UNL.
```
Comment (intelliot) [inline content/faq.md]:
```suggestion
There can be other UNLs in use on the network. The dUNL is the most widely-used UNL, but validators can choose others.
```

File: content/faq.md [hunk 3]
```diff
@@ -141,16 +147,9 @@ Once the pull request passes automated tests and receives approvals from reviewe
...
-
-You don't need to use Ripple’s version of the XRP Ledger software to interact with the XRP Ledger. `rippled` is open-source software and Ripple grants anyone the ability to use, extend, and modify it as long as they follow the terms of the [ISC license](https://github.com/ripple/rippled/blob/develop/LICENSE.md). The ISC License is very permissive compared to some other open-source licenses that strictly limit how you can extend and adapt the software.
-
-Ripple is also one of several entities who publish recommended UNL settings and runs validators on the network. As of mid-2021, Ripple runs 6 of the 35+ validators in the default UNL.
-
-
-#### Does Ripple offer a secure method to download their software?
+Ripple does contribute to a reference implementation of the core XRP Ledger server ([`rippled`](https://github.com/xrplf/rippled)) and employs a team of engineers who contribute to the open-source codebase. Ripple also periodically publishes pre-compiled binary packages of the software as a convenience. Anyone is free to [download and compile the software from source](install-rippled.html), if they prefer.
```
Comment (intelliot) [inline content/faq.md]:
```suggestion
Ripple contributes to a reference implementation of the core XRP Ledger server ([`rippled`](https://github.com/xrplf/rippled)) and employs a team of engineers who contribute to the open-source codebase. Ripple periodically publishes pre-compiled binary packages of the software for convenience. Anyone can [download and compile the software from source](install-rippled.html).
```

File: content/faq.md [hunk 4]
```diff
@@ -141,16 +147,9 @@ Once the pull request passes automated tests and receives approvals from reviewe
...
 
-`rippled` source code is available at <https://github.com/ripple/rippled>, where the tip of the `master`, `release` and `develop` branches always contains a version-setting commit signed by a `rippled` developer. Ripple also publishes pre-built binary packages for CentOS, RedHat Enterprise Linux, Fedora, Ubuntu, and Debian Linux. Those packages are digitally signed so that they are tamper-evident and their authenticity can be verified. Lastly, release bulletins are made available over a secure website, and include the commit ID of the repository, as well as the cryptographic hash values of the packages that are published. <!-- STYLE_OVERRIDE: evident -->
+Ripple is also one of several entities who publish recommended UNL settings and runs validators on the network. As of late-2022, Ripple runs 3 of the 35+ validators in the default UNL.
```
Comment (intelliot) [inline content/faq.md]:
```suggestion
Several entities publish recommended validator lists (UNLs). As of September 2022, Ripple runs 3 of the 35 validators in the default UNL.
```

File: content/faq.md [hunk 5]
```diff
@@ -101,6 +101,12 @@ Publishers of individual default UNLs set their own policies for when to add or
 For recommendations and best practices, see [Run `rippled` as a Validator](run-rippled-as-a-validator.html).
 
 
+#### If the dUNL has the most influence on the network, then is the XRPL centralized?
+Validators can choose to not use the dUNL, or any widely-used UNL for that matter. Anyone can create a new UNL at any time.
+
+There can be other UNLs in use on the network. The dUNL is the most widely-used UNL, but validators can choose others.
+
+
```
Comment (cjcobb23) [inline content/faq.md:111]: I think we should mention something about the possibility of forking when nodes use different UNLs

File: content/faq.md [hunk 6]
```diff
@@ -101,6 +101,12 @@ Publishers of individual default UNLs set their own policies for when to add or
 For recommendations and best practices, see [Run `rippled` as a Validator](run-rippled-as-a-validator.html).
 
 
+#### If the dUNL has the most influence on the network, then is the XRPL centralized?
+Validators can choose to not use the dUNL, or any widely-used UNL for that matter. Anyone can create a new UNL at any time.
+
+There can be other UNLs in use on the network. The dUNL is the most widely-used UNL, but validators can choose others.
```
Comment (mDuo13) [inline content/faq.md]:
```suggestion
There can be multiple UNLs in use on the same network. Each operator can customize their server's own UNL or choose to follow a different recommended list. All these servers can still operate on the same chain and reach consensus with one another.

However, if your UNL does not have sufficient overlap with the UNLs used by others, there is a risk that your server forks away from the rest of the network. As long as your UNL has > 90% overlap with the one used by people you're transacting with, you are completely safe from forking. If you have less overlap, you may still be able to follow the same chain, but the chances of forking increase with lower overlap, worse network connectivity, and the presence of unreliable or malicious validators on your UNL.
```

**Review by mDuo13 — APPROVED** [review]
Suggested a change to address @cjcobb23's suggestion regarding overlap. Otherwise, these changes look good to me.

**Comment by wojake** [comment]
Same here, LGTM

**Pull request merged**